components/style/properties/longhand/position.mako.rs

```diff
@@ -157,10 +157,8 @@ ${helpers.predefined_type("flex-basis",
 
     // min-width, min-height, min-block-size, min-inline-size
     ${helpers.predefined_type("min-%s" % size,
-                              "LengthOrPercentage",
-                              "computed::LengthOrPercentage::Length(Au(0))",
-                              "parse_non_negative",
-                              needs_context=False,
+                              "MinLength",
```

Comment:
You should have an `if product == "gecko"` that changes the type of this.

Servo mode should still use LengthOrPercentage. Same for the next line.

This should build in both `./mach build-geckolib` and `./mach build -d` mode.